config/cortex/values.yaml

```diff
@@ -229,6 +241,7 @@ config:
       kvstore:
         store: memberlist
   compactor:
+    block_deletion_marks_migration_enabled: false
```

Comment:
Are we sure this option is a good idea to have by default?

> "This option can (and should) be safely disabled as soon as the compactor has successfully run at least once."

https://cortexmetrics.io/docs/blocks-storage/compactor/